programmers/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,24 @@
+function solution(lottos, win_nums) {
+    
+    let zeroNum = 0;
+    let matchNum = 0;
+    
+    for (let i = 0; i < 6; i++) {
+        if(lottos[i] === 0) zeroNum += 1; // lottos배열에서 0의 갯수를 셈하여 zeroNum에 더해줌
+        
+        for (let j = 0; j < 6; j++) {
+            if (lottos[i] === win_nums[j]) {
+                matchNum += 1; // lottos배열에서 win_nums배열원소들과 비교해서 맞으면 더해줌
+                break; //  맞는 값을 찾으면 더 반복 할 필요가 없으므로 바깥for문으로 이동
+            }
+        }
```

Comment:
```suggestion
 if (win_nums.includes(lottos[i])) matchNum++;
```
배열안에 같은 값을 찾아 boolean값으로 반환하는 includes 메서드를 이용하면 간결하게 표현이 가능해요!